src/ResourceManager/IotHub/Commands.IotHub/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.IotHub" version="1.1.2" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.IotHub" version="1.1.3" targetFramework="net452" />
```

Comment:
Please revert the three psm1 files below this file - these are added at build but should not be pushed in a PR

Reply:
Reverted